jwst/fringe/fringe.py

```diff
@@ -77,12 +75,11 @@ def apply_fringe(input_model, fringe):
 ###    will be logged for now. This behavior may be changed later.
 ###
     # set DQ flag for bad pixels in the fringe
-#       dq_mask = fringe_dq * 0
-#       dq_mask[np.where(fringe_dq != 0)] = dqflags.pixel['DEAD']
+#   dq_mask = fringe.dq * 0
```

Comment:
I am just curious I would of thought that if the fringe_dq !=0 then the dq_mask would of been set to NON_SCIENCE rather than DEAD.

Reply:
No idea. That's just a holdover from the original code in the step, along with the commentary that it's all deactivated for now, because the DQ values in the reference file don't actually correspond to bad pixels (or some such). You'll need to talk with the original developer of the step and ref file to sort it out. Now, who could that be ...